### ISSUE TITLE
0.11.0 fails if it cannot figure out a git version

### ISSUE THREAD

**Issue by seancorfield** [liquidz/antq#58]
Running this on our codebase at work produces:
```
The tag for current version is not found: https://github.com/bcgit/bc-java/
Execution error (NullPointerException) at antq.util.maven/get-scm-url (maven.clj:97).
Cannot invoke "org.apache.maven.model.Scm.getUrl()" because "scm" is null
```
(I think this must be a transitive dependency that something brings in)

Here's the top of the stack trace for more context:
```
  [[antq.util.maven$get_scm_url invokeStatic "maven.clj" 97]
   [antq.util.maven$get_scm_url invoke "maven.clj" 95]
   [antq.diff.java$get_scm_url_STAR_ invokeStatic "java.clj" 60]
   [antq.diff.java$get_scm_url_STAR_ invoke "java.clj" 52]
```

**Comment by seancorfield** [comment]
I suspect this code should use `some->` for at least some parts: https://github.com/liquidz/antq/blob/ffaa6abbfb6305f264bc71de595326d8803454a8/src/antq/diff/java.clj#L58-L65

**Comment by seancorfield** [comment]
I suspect this code should use `some->` for at least some pieces: https://github.com/liquidz/antq/blob/ffaa6abbfb6305f264bc71de595326d8803454a8/src/antq/diff/java.clj#L58-L65

**Comment by liquidz** [comment]
@seancorfield Thanks!
I fixed in `dev` branch. Could you try?

**Comment by liquidz** [comment]
I've released v0.11.1 for now.
